docs/iris/src/whatsnew/contributions_1.9/newfeature_2015-08-18_strict_grib1_load.txt

```diff
@@ -0,0 +1,2 @@
+* Support for GRIB1 messages has been added to the new strict_grib_load approach.  This uses 
+  the legacy metadata interpretation code, enabling GRIB1 loading to continue to be supported.
```

Comment:
I wouldn't make any reference to the implementation, if we can avoid it !
Is it perhaps necessary to add some caveats though, like "not all templates are supported" ?

I am also concerned that load callbacks for GRIB1 messages will now be incompatible with the GRIB2 form -- I intend to look into this + make a general comment about it later ...